Dockerfile

```diff
@@ -1,22 +1,22 @@
 #
 # Dockerfile for Storm Mesos framework
 #
-FROM mesosphere/mesos:0.22.0-1.0.ubuntu1404
+FROM mesosphere/mesos:0.25.0-0.2.70.ubuntu1404
 MAINTAINER Timothy Chen <tnachen@gmail.com>
 
 # build packages
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update
-RUN apt-get install -yq openjdk-7-jdk
+RUN apt-get install -yq openjdk-7-jdk maven wget
 
 # export environment
 ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
-# copy local checkout into /opt/storm
-ADD . /opt/storm
+ADD . /work
 
+WORKDIR /work
+RUN ./bin/build-release.sh main
```

Comment:
Sorry, but I'm still a bit lost, I blame my inexperience with Docker.  My understanding is that a Docker image should (must?) have a "thing" to be run when the image's container launches, where this "thing" is specified via either ENTRYPOINT or CMD.  But I don't see a CMD either.

I guess this implies that when we run the resultant Docker image we now *must* specific the cmd to be executed (somehow).  However, I don't see any docs in this PR about how to do this for this repo (meaning both the docker syntax and the image-resident cmd to be run).  Can you please add those?

(BTW, after overcoming my initial forgetfulness above about the purpose of the Dockerfile, I totally support this pulling of the `build-release.sh` invocation into the Dockerfile, instead of having it be a separate external step.)